events-api/src/modules/events/events.controller.ts

```diff
@@ -47,4 +47,45 @@ export class EventsController {
       data: foundEvent,
     };
   }
+
+  @Patch(':eventId')
+  @Roles(UserRole.Admin)
+  @UseGuards(AuthTokenGuard, RolesGuard)
+  public async updateEvent(
+    @Body() dto: UpdateEventReqDto,
+    @Param(
+      'eventId',
+      new ParseUUIDPipe({
```

Comment:
I think that we can move it to the separate class for readability

Reply:
fixed